### DISCUSSION TITLE
About the road coordinates in the sample dataset

### DISCUSSION THREAD

**Discussion by ali-ihsan-gullu** [christianbirchler-org/sdc-testing-competition#22] [Q&A] [answered]
Hello. We are expecting the data in the same format as the sample dataset for the competition. Is it possible for the number of road coordinates to change, or will we be provided with exactly 197 (x, y) road points for each test case in the competition as well?

Thank you.

**Comment by ChristianBirchler** [comment, marked as answer]
Hi @vatozZ , 
thanks for the question.

You should assume that the number of road points can vary per test case.

Btw, the deadline got extended by a few days (November 17).

Best, Christian
